src/Filtering/ImageStatistics/CalculateImageMoments/Code.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+# Copyright NumFOCUS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import itk
+import argparse
+
+if len(sys.argv) != 2:
+    print("Usage: " + sys.argv[0] + "<InputFileName>")
```

Comment:
Is this usage dissonant with `argparse`?

Reply:
Yeah, it's not using ```argparse``` at all, as far as I can tell.  This is from the cookiecutter template though:
https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/blob/fd2b39b4d652d89e9a7de53ea1c947cd8a597c30/Utilities/CookieCutter/%7B%7Bcookiecutter.example_name%7D%7D/Code.py

Reply:
Could you also fix cookiecutter template too? 🥺🙏

Reply:
Sure thing, I'll include the C++/Python reordering too once I get the go ahead.

Reply:
@thewtex @dzenanz So, it's very simply to switch over to arparse usage (we just delete that if-block).  However, it will return a status code of 2 on error instead of 1 right now.  Is that an issue?

Reply:
No issue at all.

Reply:
Great, it's possible to change the exit code, but then you have to add your own try/except block to overwrite the default.